packages/sanity/src/core/create/__telemetry__/create.telemetry.ts

```diff
@@ -0,0 +1,32 @@
+import {defineEvent} from '@sanity/telemetry'
```

Comment:
cc @sanity-io/data-eng for review

Reply:
The Create side equivalents will land in a couple of days fyi

Reply:
@snorrees These seems to follow the copy of the buttons directly. Could we instead make them more about the underlying action? Perhaps instead of "Start In Create Clicked" we can do "Author In Create Clicked" or along those lines

Reply:
should all be fixed now @svirs